marklogic-data-hub/src/main/resources/ml-modules/root/marklogic-data-hub-spark-connector/initializeRead.sjs

```diff
@@ -20,7 +20,7 @@ const readLib = require("readLib.sjs");
 
 /**
  * Build an Optic plan based on the user's inputs.
- * 
+ *
```

Comment:
Undo this change just to avoid making any changes to this file (which complicates the version history of the file).